ecosystem/html_templates/webpage.html.jinja

```diff
@@ -8,6 +9,19 @@
     />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>{{ header.title.bold }} {{ header.title.normal }}</title>
+    <link
```

Comment:
Maybe comment why these two CSS files are only for standalone.

Reply:
Done in 35eb19256065b4b09bfe4b76acde0b3663ab093b, thanks.